### PR TITLE
Use real Error in tests.

### PR DESCRIPTION
**Pull request by jmdobry**
First of several "mini" PRs

### CODE REVIEW

File: bigquery/test/queries.test.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ describe('bigquery:query', function () {
     });
 
     it('should handle error', function () {
-      var error = Error('syncQueryError');
```
Comment (ace-n) [inline bigquery/test/queries.test.js]: Should this be a global variable? (Or do we have to create a new one within each test?)

File: logging/test/logs.test.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ describe('logging:entries', function () {
     });
 
     it('should handle error', function () {
-      var error = 'error';
```
Comment (ace-n) [inline logging/test/logs.test.js]: Global variable?

File: logging/test/sinks.test.js [hunk 3]
```diff
@@ -149,7 +149,7 @@ describe('logging:sinks', function () {
     });
 
     it('should handle error', function () {
-      var error = 'error';
```
Comment (ace-n) [inline logging/test/sinks.test.js]: Global variable?

**Comment by ace-n** [comment]
Can some of these be converted to one-per-file global variables?

**Comment by jmdobry** [comment]
How about doing something like that in a separate "Use per-file global variables in tests" PR?

**Comment by ace-n** [comment]
Hmm - that could work, though IMHO the two are related enough to put in the same PR (since they're changing the same things).

**Comment by jmdobry** [comment]
I was thinking there are other variables (not error vars) that could benefit from going global, hence a separate PR.

**Pull request merged**